bundle.yaml

```diff
@@ -669,9 +669,14 @@ spec:
                   The secret is mounted into /etc/alertmanager/config.
                 type: string
               containers:
-                description: Containers allows injecting additional containers. This
+                description: 'Containers allows injecting additional containers. This
```

Comment:
Curious but how come this wtih `'`?

Reply:
hmm not sure :)

Reply:
Maybe because I've updated the description and it now contains backticks?

Reply:
No clue, the only thing I am concerned and why I asked is because if its still valid as we don't have it for other descriptions. But I guess e2e tests passed so that confirms it?